DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -72,15 +72,6 @@
           "revision": "9483a5d459b45c3ffd059f7b55f9638e268632fd",
           "version": "1.5.0"
         }
-      },
-      {
-        "package": "TrackerRadarKit",
-        "repositoryURL": "https://github.com/duckduckgo/TrackerRadarKit.git",
-        "state": {
-          "branch": null,
-          "revision": "5f4caf35b8418700a48c64c7c61eb43308c8dacc",
-          "version": "1.0.3"
-        }
```

Comment:
This is regrettably Xcode working normally, it adds and removes this block on a whim. It's harmless, but I can forcibly reset this change if you want.

Reply:
Since we're not pinned to a specific version, could this backfire (e.g. we without realising end up upping to 1.0.4 and that breaks something)

Reply:
Fortunately we'll be good here before we merge:

* BSK points to an exact version of TrackerRadarKit [here](https://github.com/duckduckgo/BrowserServicesKit/blob/main/Package.swift#L18)
* We do point to an exact version of BSK in the macOS app, just that this branch is pointing to a BSK branch for testing - before this is merged though, this will be updated to point to an exact BSK version